spring-data-relational/src/main/java/org/springframework/data/relational/core/conversion/AbstractRelationalConverter.java

```diff
@@ -89,7 +89,7 @@ public ConversionService getConversionService() {
 		return conversionService;
 	}
 
-	public CustomConversions getConversions() {
+	public CustomConversions getCustomConversions() {
```

Comment:
This is a breaking change.

Reply:
I agree. Idk, I just renamed it for readability, to not confuse it in the code with `ConversionService`. Do you think it should be rolled back, or we're going to introduce this pr in the next major release, so it can wait?

Reply:
Please do not introduce breaking changes if you're looking for inclusion in a minor release.